prow/bump.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if [ "$#" -ne 1 ]; then
+  echo "usage: $0 [program]"
+  exit 1
+fi
+
+cd $(dirname $0)
+
+makefile_version_re="^\(${1^^}_VERSION.*=\s*\)"
+version=$(sed -n "s/$makefile_version_re//p" Makefile)
```

Comment:
BOO. Just freaking edit the file. The complexity of this script is not work the microscopic time savings.